packages/paste-website/src/components/icons-list/IconsList.tsx

```diff
@@ -0,0 +1,249 @@
+        return {
+          logos: [...prev.logos, Icon],
+          products: [...prev.products],
+          ui: [...prev.ui],
+        };
+      }
+      return {
+        logos: [...prev.logos],
+        products: [...prev.products],
+        ui: [...prev.ui, Icon],
+      };
+    },
+    {logos: [], products: [], ui: []}
+  );
+
+const getFirstIcon = (iconsList: GroupedList): IconObject | null => {
+  if (iconsList.ui.length !== 0) {
+    return iconsList.ui[0];
+  }
+  if (iconsList.products.length !== 0) {
+    return iconsList.products[0];
+  }
+  return null;
+};
+
+// Our Grid Column doesn't allow position for sticky, but our box doesn't allow percentage widths
+// Styled Box is fine I guess, but we might want to expand Box to take any width
+const IconColumn = styled(Box)({
+  width: '41.66666666666667%',
+});
+
+const IconsList: React.FC<IconsListProps> = () => {
+  const {isPasteTheme} = React.useContext(SiteBodyContext);
+  const filterID = useUID();
+  const iconKeySeed = useUIDSeed();
+  const groupedIconList = getGroupedList(iconsJson);
+  const [filterString, setFilterString] = React.useState('');
+  const [iconsList, setIconsList] = React.useState(groupedIconList);
+  const [selectedIcon, setSelectedIcon] = React.useState(getFirstIcon(iconsList));
+  const uiComposite = useCompositeState();
+  const productComposite = useCompositeState();
+  const logoComposite = useCompositeState();
+  return (
+    <Box maxWidth="size120">
+      <Grid gutter="space60">
+        <Column span={7}>
+          <Box as="form" role="search" onSubmit={e => e.preventDefault()} marginBottom="space70">
+            <FormLabel htmlFor={filterID}>Filter Icons</FormLabel>
+            <FormInput
+              autoComplete="off"
+              id={filterID}
+              type="text"
+              value={filterString}
+              placeholder="Filter by name..."
+              onChange={e => {
+                const filteredList = iconsJson.filter(({name}: IconObject) =>
+                  name.toLowerCase().includes(e.currentTarget.value.toLowerCase())
+                );
+                const filteredGroupedList = getGroupedList(filteredList);
+                setIconsList(filteredGroupedList);
+                setSelectedIcon(getFirstIcon(filteredGroupedList));
+                setFilterString(e.currentTarget.value);
+              }}
+            />
+          </Box>
+        </Column>
+        <Column span={7}>
+          {iconsList.ui.length > 0 && (
+            <>
+              <Heading as="h2" variant="heading20">
+                UI Icons
+              </Heading>
+              <Paragraph>
+                UI icons are the general collection of icons that can be used in most user interfaces. They are most
+                commonly used in conjunction with performing an interaction. Be sure to follow the{' '}
+                <SiteLink to="/icons/usage-guidelines">usage guidelines</SiteLink> carefully when placing an icon in
```

Comment:
1. I would move this note of "usage" guidelines" to under the Icon List heading.
2. Before this guideline sentence, add in the Icon List subtitle "UI, Twilio Product, and Logo icons that are available in the Paste icon system"

Reply:
Also add the "Be sure to read the engineering getting started page to ensure you have the correct dependencies." from icon card in here.